src/test/resources/tcp-connection-timeout-config.xml

```diff
@@ -10,7 +10,7 @@
     <sockets:request-config name="tcp-requester">
         <!--Non-rutable IP address -->
         <sockets:tcp-requester-connection host="10.255.255.255" port="${port}"
```

Comment:
The comment has a typo, please add the 'o' in 'routable' :)

Reply:
Not mine change, but fixed.